src/GitVersion.Core.Tests/Configuration/ConfigProviderTests.CanWriteOutEffectiveConfiguration.approved.txt

```diff
@@ -18,7 +18,7 @@ branches:
     tag: alpha
     increment: Minor
     prevent-increment-of-merged-branch-version: false
-    track-merge-target: true
+    track-merge-target: false
```

Comment:
Why the change in `track-remote-target` here?

Reply:
track-merge-target was not used. Anyway I have changed it because I thought this should be the default behavior. I think for the release 5.x we need to set this property for sure for support and master and maybe for develop branch as well (please see the discussion 3052).